Sources/Charcoal/Filters/Range/Input/NumberInputView.swift

```diff
@@ -257,20 +257,20 @@ extension NumberInputView: UITextFieldDelegate {
 private struct Style {
     static let textColor: UIColor = .licorice
     static let errorTextColor: UIColor = .cherry
-    static let hintNormalFont: UIFont? = UIFont(name: FontType.light.rawValue, size: 16)
-    static let hintActiveFont: UIFont? = UIFont(name: FontType.medium.rawValue, size: 16)
+    static let hintNormalFont: UIFont = .body
+    static let hintActiveFont: UIFont = .bodyStrong
```

Comment:
Just a hint, the names _might_ change yet again😞 Got some super secret inside info from @osanoj that the designers are discussing the namings.